tests/integration/routes/test_questionnaire.py

```diff
@@ -53,3 +53,21 @@ def test_options_request_before_request(self):
 
         for output in logs.output:
             self.assertNotIn("questionnaire request", output)
+
+    def test_get_request_logs_output(self):
+        self.launchSurvey("test_hub_and_spoke")
+        with self.assertLogs() as logs:
+            self.get("/questionnaire/")
+            self.assertStatusOK()
+
+            request_log = logs.output[0]
+            questionnaire_request_log = logs.output[1]
+
+            self.assertNotIn("tx_id", request_log)
```

Comment:
This should also assert things that should be in request log.

Reply:
Added, thanks!